docs/core-concepts/atomic-swap-htlc.md

```diff
@@ -0,0 +1,146 @@
+---
+id: atomic-swap-htlc
+title: Atomic Swaps using HTLCs
+sidebar_label: Atomic Swaps using HTLCs
+---
+
+This documentation section provides an introduction to atomic swaps, hash time locked contracts (HTLCs) and how these concepts are used in COMIT.
+Atomic Swaps and HTLCs are core concepts of the COMIT protocol.
+
+<iframe 
+    src="https://docs.google.com/presentation/d/e/2PACX-1vRjJeto_eQaZ8k5Ct00UzceHaSs-uIuvPmTFV6jP9SP6N-EKmct8H4cfN3xa2h-RFs8Gm4Gia41Je7m/embed?start=true&loop=false&delayms=1000" 
+    frameborder="0" 
+    width="801" 
+    height="480" 
+    allowfullscreen="true"
+    mozallowfullscreen="true" 
+    webkitallowfullscreen="true">
+</iframe>
+
+This slide shows an overview of how atomics swaps using HTLCs work in COMIT, more details can be found in the sections below.
+
+## Atomic swap
+
+An Atomic Swap is an exchange of two digital assets between two parties in which both parties are guaranteed to either receive the other asset OR retain their original asset i.e., an atomic swap guarantees that neither party is able to take both assets.
+
+Atomicity of the swap can be achieved by [HTLC](#hash-time-locked-contract-htlc)s that serve as a programmable escrow, or other concepts as explained in the [Bitcoin Wiki](https://en.bitcoin.it/wiki/Atomic_swap).
+
+## Hash Time Locked Contract (HTLC)
+
+HTLC stands for Hash Time Locked Contract. An HTLC is a script or smart contract that locks a digital asset over time.
+HTLCs require a ledger that provides a time lock.
+
+Both parties are required to have the same knowledge of all parameters for the HTLCs on both ledgers before they can start.
+Both parties then have the same functions available within the HTLCs and can decide to move forward or wait to take their money back. 
+No party has access to both assets at the same time within the time boundaries of the atomic swap. 
+
+HTLCs typically define three functions:
+
+1. **Fund**: Locking up the asset in the HTLC.
+1. **Redeem**: Until the expiry time-stamp is reached the other party has the chance to take the locked-up assets with the secret.
+1. **Refund**: After the expiry condition has been met, the party that locked up the asset can take it back.
+
+HTLCs are secured by a time-lock and hash-lock.
+The hash-lock requires the generation of a secret prior to funding. 
+The order of the actions is then tied to the cryptographic roles of Alice Bob. 
+Alice is the role that comes up with the secret.
+She shares the hash of that secret with Bob enabling him to fund on his side.
+The party in the cryptographic role of Alice has to fund and redeem first, because she has the actual secret.
+
+## Atomic Swaps using HTLCs in COMIT
```

Comment:
I think this section should be relabelled as 'Negotiation' instead.

Reply:
The original heading can be assigned to the paragraphs after it.

Reply:
It seems that the first sentence of this section, mentioning `Negotiation` is confusing. It can be removed. I put it there to have a link to the negotiation phase to try to create a bit more context.